package.json

```diff
@@ -2,34 +2,47 @@
   "name": "webhook-handler",
   "version": "1.0.0",
   "scripts": {
-    "build": "ENVIRONMENT=local npm run build:config && sls webpack",
-    "build:config": "tsc src/config.ts",
-    "clean": "skripts clean scripts --pattern '*.js'",
-    "deploy": "npm run build:config && sls deploy && npm run update:all",
-    "format": "skripts format",
-    "jest": "PARTNER_QUEUE_URL=partner.com RESULT_QUEUE_URL=result.com ERROR_QUEUE_URL=error.com VERSION=v1 jest",
-    "lint": "skripts lint --fix",
+    "build": "export ENVIRONMENT=local && npm run build:config && sls webpack",
+    "build:config": "tsc --project ./tsconfig.json",
+    "deploy": "npm run build:config && sls deploy",
+    "format": "prettier --ignore-path .prettierignore --write \"./**/*.+(js|jsx|ts|tsx|json|yml|yaml|md|html|css|less|scss|graphql)\"",
+    "jest": "PARTNER_QUEUE_URL=partner.com RESULT_QUEUE_URL=result.com ERROR_QUEUE_URL=error.com VERSION=v1 jest --silent --verbose",
+    "lint": "eslint --fix --config .eslintrc --ext '.ts,.tsx,.js,.jsx' '.'",
     "start": "cd scripts && tsc && node genEvent.js && node --inspect server.js",
-    "test": "npm run build:config && npm run lint && npm run jest && sls package",
+    "test": "export ENVIRONMENT=devint && npm run build:config && npm run lint && npm run jest && sls package",
```

Comment:
why devint instead of local here?

Reply:
I was just matching what was in the dwolla ci build step to make sure test would run locally and not impact the ci build.